### DISCUSSION TITLE
Пишет что системе Windows требуется драйвер с цифровой подписью

### DISCUSSION THREAD

**Discussion by AntonBiomusorov** [Flowseal/zapret-discord-youtube#1124] [Show and tell]
Пытался помочь подруге с ютубом и дискордом ибо у неё не фурычило всё это. сказал мол "вот скачай запретку и не беспокойся". она установила зипку, распаковала и ей пишет
![photo_5440813937659604852_x](https://github.com/user-attachments/assets/460faac5-cff4-46f0-9e0e-225e483e073e)
вот че ей делать?  просто у меня запрет установлен тот же самой, но при этом подобное не вылезало и ваще гладенько всё работало

**Comment by Flowseal** [comment]
Проблема из-за Windows7
Можете попробовать заменить файлы в папке bin на эти: https://github.com/bol-van/zapret-win-bundle/tree/master/win7

Также:
https://github.com/Flowseal/zapret-discord-youtube/issues/825

**Comment by mlatushko** [comment]
(для прочих случаев)
```
Системе Windows требуется драйвер с цифровой подписью.
Отключить проверку подписи цифровых драйверов (Windows 7)

> Win+R >> gpedit.msc [Редактор локальной групповой политики]
(Политика "Локальный компьютер") > Конфигурация пользователя
> Административные шаблоны > Система > Установка драйвера
> > Цифровая подпись драйверов устройств
(было: ‹Не задано›) >> Включить

+
[Параметры]
Если Windows обнаруживает файл драйвера без ц.п.:
>> Предупредить
```
